packages/node-fetch/tests/URL.spec.ts

```diff
@@ -22,4 +52,12 @@ describe('URL', () => {
     expect(url.hostname).toBe('[::1]');
     expect(url.port).toBe('8000');
   });
+  it('parses query strings with ports without pathname', () => {
```

Comment:
this can't be shared with other environments?

Reply:
Other environments don't use `node-fetch`. `node-fetch` package is only used on Node. No need to test this on other environments.